libyul/optimiser/StackToMemoryMover.h

```diff
@@ -52,6 +55,16 @@ namespace solidity::yul
  *     let c := _3
  *     let a := _1
  *
+ * In case f has return parameters that are moved to memory, fewer variables are returned and the return values read
+ * from memory instead. Assume the third return parameter of f (i.e. c) has to be moved to memory:
+ *     let a, b, c, d := f()
+ *   then it is replaced by
+ *     let _1, _2, _4 := f()
+ *     mstore(<memory offset for d>, _4)
```

Comment:
Why do you need this?

Reply:
Ah, this s just a continuation of the previous example?

Reply:
Yes, it can happen that both `f` moves one of its return variable to memory and the call site needs one of the variables moved to memory.